api/Element.json

```diff
@@ -7005,10 +7091,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/scroll",
           "support": {
             "chrome": {
-              "version_added": "45"
+              "version_added": "61"
```

Comment:
Based on http://mdn-bcd-collector.appspot.com/tests/api/Element/scroll this change could be because `scroll` was initially on `HTMLElement`. But I confirmed with https://software.hixie.ch/utilities/js/live-dom-viewer/saved/8714 that it was added in Chrome 61.